superduperdb/db/base/db.py

```diff
@@ -575,6 +705,36 @@ def _build_task_workflow(
                     f'{model}.predict({key})',
                 )
 
+        if s.CFG.self_hosted_vector_search:
+            return
+
+        vi_lookup = {
+            identifier: self.metadata.get_indexing_listener_of_vector_index(identifier)
+            for identifier in self.show('vector_index')
+        }
+
+        for identifier in vi_lookup:
```

Comment:
can we create a base method for _build_taskworkflow and _build_delete_taskworkflow? @blythed